demisto_sdk/commands/validate/initializer.py

```diff
@@ -571,13 +575,23 @@ def get_items_status(
                     statuses_dict[path] = git_status
             elif PACKS_PACK_META_FILE_NAME in path_str:
                 statuses_dict[path] = git_status
-            elif self.is_pack_item(path_str):
+            if (
+                self.is_pack_item(path_str)
+                or find_type(path_str) not in SKIP_RELEASE_NOTES_FOR_TYPES
+            ):
```

Comment:
Maybe we should add this condition at the end of the for loop after the else statement?
The idea of this else statement is to catch items that doesn't have multiple files (like mappers, incident fields, etc...) so I think we will miss them in the current implementation.

Reply:
not sure I follow..
The implementation of is_pack_item returns True if the file is one of the following:
`PACKS_PACK_IGNORE_FILE_NAME,
RELEASE_NOTES_DIR,
PACKS_WHITELIST_FILE_NAME,
PACKS_README_FILE_NAME,
AUTHOR_IMAGE_FILE_NAME,
PACKS_CONTRIBUTORS_FILE_NAME,
DOC_FILES_DIR,`

The else refers to the following conditions:
if f"/{INTEGRATIONS_DIR}/" in path_str or f"/{SCRIPTS_DIR}/" in path_str: ...
elif f"/{PLAYBOOKS_DIR }/" in path_str: ...
elif MODELING_RULES_DIR in path_str or PARSING_RULES_DIR in path_str:...
elif PACKS_PACK_META_FILE_NAME in path_str:...

Correct me if I'm wrong but if any of these conditions hold is_pack_item will return False anyway, so no harm in making it an if statement instead of an elif

Reply:
good catch